nav2_bt_navigator/behavior_trees/follow_waypoints.xml

```diff
@@ -0,0 +1,23 @@
+
+<!--
+  BT to process FollowWaypoints.action
+-->
+<root main_tree_to_execute="MainTree">
```

Comment:
This needs documentation added and a diagram of this tree to the readmes

Reply:
Added to `Future work` of description.